OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -127,6 +124,65 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         }
     }
 
+    public void setupToolbar() {
+        AppBarLayout appbar = getActivity().findViewById(R.id.appbar);
+
+        collapsedToolbar = getActivity().findViewById(R.id.collapsed_toolbar_layout);
```

Comment:
IMO, better to move the toolbar layout within the Fragment instated of Activity, cuz there is not common layout in the toolbar.

Reply:
We can perform this change as part of [LEARNER-8921](https://2u-internal.atlassian.net/browse/LEARNER-8921).